README.md

```diff
@@ -7,7 +7,7 @@
 
 <img width="300px" src="https://user-images.githubusercontent.com/209884/67159116-64d94b80-f37b-11e9-9b28-f8379636a43c.png"></img>
 
-# Why a new library?
+## Why a new library?
```

Comment:
I don't like to change the header .
Please keep the markdown structure overall .

Reply:
It's still a header, it can still be referenced the same way `http://<link-to-readme>#why-a-new-library, it's just header level 2 since `# YAML support for the Go language` at the top is already level 1. 

But no problem I'll change back.